lib/wiki_course_output.rb

```diff
@@ -90,14 +93,16 @@ def course_week(week, week_number)
       week_output += content_block(block)
     end
 
-    week_output += "{{end of course week}}\r"
+    week_end_template = @templates['templates']['end_of_week']
```

Comment:
Instead of a method in the other class, we should make a module, like `WikiOutputTemplates`, so we can reuse it here.